modules/nw-networkpolicy-about.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// networking/configuring-networkpolicy.adoc
+
+[id="nw-networkpolicy-about-{context}"]
+
+= About network policy
+
+In {product-title} {product-version} `NetworkPolicy` is the default mode for the
+`OpenShiftSDN` plug-in for the Kubernetes Container Network Interface (CNI).
+
+[NOTE]
+====
+The `v1` NetworkPolicy features are available only in {product-title}. This
+means that egress policy types, IPBlock, and combining `podSelector` and
+`namespaceSelector` are not available in {product-title}.
+====
+
+In a cluster network isolation is controlled entirely by
+link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/network/network-policy.md[`NetworkPolicy`
+objects]. By default, all pods in a project are accessible from other pods and
+network endpoints. To isolate one or more pods in a project, you can create
+`NetworkPolicy` objects in that project to indicate the allowed incoming
+connections. Project administrators can create and delete `NetworkPolicy`
+objects within their own project.
+
+Pods that do not have `NetworkPolicy` objects pointing to them are fully
+accessible, whereas, pods that have one or more `NetworkPolicy` objects pointing
+to them are isolated. These isolated pods only accept connections that are
+accepted by at least one of their `NetworkPolicy` objects.
+
+Following are a few sample `NetworkPolicy` object definitions supporting
+different scenarios:
+
+* *Deny All Traffic*
```

Comment:
I don't think we use asterisks except for referring to GUI items. Suggest instead trying this syntax:

```
Deny all traffic::
```

Also, these should all be sentence case.